fleece/cli/build/build.py

```diff
@@ -104,29 +128,67 @@ def build(args):
     with open(requirements_path, 'rb') as fp:
         dependencies_sha1 = hashlib.sha1(fp.read()).hexdigest()
 
+    # Set up volumes
+
+    src = docker_api.containers.create(
+        'alpine:3.4',
+        '/bin/true',
+        volumes=[
+            '/src',
+            '/requirements',
+            '/dist'
+        ]
+    )
+
+    stream = BytesIO()
+
+    with tarfile.open('src.tar', mode='w') as tar:
+        tar.add(src_dir, arcname=os.path.sep)
+
+    with tarfile.open(fileobj=stream, mode='w') as tar:
+        tar.add(src_dir, arcname=os.path.sep)
+    stream.seek(0)
+    src.put_archive(data=stream, path='/src')
+
+    stream_1 = BytesIO()
+
+    with tarfile.open('req.tar', mode='w') as tar:
+        tar.add(requirements_path, arcname='requirements.txt')
```

Comment:
Does this req.tar have a purpose or is it left over from an earlier version of the code?

Reply:
Actually nm, wasn't looking at the last commit.

Reply:
Hah no worries. That was there originally for testing purposes. I'm planning on squashing this down once everything looks good.